src/document-click.js

```diff
@@ -108,6 +113,21 @@ export function onDocumentElementClick_(e, viewport) {
   const win = doc.defaultView;
 
   const tgtLoc = parseUrl(target.href);
+
+  // On Safari iOS, custom protocol links will fail to open apps when the
+  // document is iframed - in order to go around this, we set the top.location
+  // to the custom protocol href.
+  const isSafariIOS = platform.isIos() && platform.isSafari();
```

Comment:
I'd assume the same failure would be observed on iOS Chrome. Could you test there as well?